uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard-fields.html

```diff
@@ -9,7 +9,8 @@
         </help-text>
       </label>
       <div ng-class="{ 'col-md-9': !field.labelLayout, 'col-md-12': field.labelLayout === 'horizontal' }">
-        [template of the field]
+        <ng-include
```

Comment:
Considering how we **love** to add custom stuff like tooltips, hints etc to fields, I would keep it it simple and would not add another prop to the map. The markup for the checkbox is as per the styleguide, it is not custom, so I would rather have a couple of "dupped" templates rather than making this component even more complicated.